### DISCUSSION TITLE
What does "vAPI" stand for?

### DISCUSSION THREAD

**Discussion by aaronliu0130** [uBlockOrigin/uBlock-issues#3114] [Q&A] [answered]
I've recently encountered an extension that bundles the entirety of μBlock just for its buttons, and all it actually uses is stuff from uDom and vAPI. Now, I'm curious as to what the "v" stands for.

**Comment by iam-py-test** [comment]
> I've recently encountered an extension that bundles the entirety of μBlock

Slightly off-topic, but I am curious; what extension? [Are they following the uBlock Origin license](https://github.com/gorhill/uBlock/blob/master/LICENSE.txt)?
Thanks

**Reply by aaronliu0130** [reply]
I'm not sure if they follow it, since one can view the extension source code through going to "Extensions/gibberish" under the profile folder of the browser. The folder doesn't have a LICENSE file attached though, but the used source code remains intact.

It's Epic Privacy Browser's "Epic Proxy" extension. It's not available for public download, so you'll have to install the browser and then go under extension ID jbblmbimjemfdipkoiioihcopmhkeldf's folder. All of this is just to make an options page for whitelisting. I was trying to port it to Firefox this afternoon.

**Reply by iam-py-test** [reply]
Thanks
I was able to find it bundled in their offline installer (crx below, extract the .crx file like a ZIP file and you can see the contents)
[Epic Encrypted Proxy.zip](https://github.com/uBlockOrigin/uBlock-issues/files/14230537/Epic.Encrypted.Proxy.zip)

**Comment by uBlock-user** [comment, marked as answer]
vendor API
